web/annotation_editor_layer_builder.css

```diff
@@ -132,6 +133,7 @@
 
 .annotationEditorLayer .inkEditor.editing {
   resize: none;
+  cursor: url(images/toolbarButton-editorInk.svg) 0 16, pointer;
```

Comment:
Also, to improve maintainability (similar to the `viewer.css` file):
```suggestion
  cursor: var(--editorInk-editing-cursor) 0 16, pointer;
```

... and add the following to the `:root`-declaration at the top of this file:
```css
--editorInk-editing-cursor: url(images/toolbarButton-editorInk.svg);
```